pkg/autohealer/wflexectracker.go

```diff
@@ -99,17 +102,26 @@ outerloop:
 }
 
 func (tracker *WFLExecTracker) executeWFL() error {
+	var msg []interface{}
 	tracker.numRetried++
 	if tracker.numRetried > tracker.maxRetries {
-		level.Debug(tracker.logger).Log("msg", "Retried workflow executions exceeds maxRetries",
-			"workflow", tracker.mistralAct.WorkflowID)
+		msg = common.CnvSliceStrToSliceInf(append([]string{
+			"msg", "Retried workflow executions exceeds maxRetries"},
+			tracker.mistralAct.InfoLog()...))
+		level.Debug(tracker.logger).Log(msg...)
 		return errors.Errorf("number of retried reached maximum")
 	}
 	exec, err := alert.ExecuteWorkflow(tracker.os, &tracker.mistralAct)
 	if err != nil {
-		level.Error(tracker.logger).Log("msg", "Error while executing workflow", "err", err)
+		msg = common.CnvSliceStrToSliceInf(append([]string{
+			"msg", "Exec action failed",
+			"err", err.Error()},
+			tracker.mistralAct.InfoLog()...))
+		level.Error(tracker.logger).Log(msg...)
+		exporter.ReportFailureHealerActionCounter(cluster.GetID(), "mistral")
```

Comment:
I reported at https://github.com/vCloud-DFTBA/faythe/blob/master/pkg/autohealer/healer.go#L316.

I consider 1 Mistral Action as a sequence of retries if exist, not a retry itself. By this way, for 1 Mistral processing there is only 1 either failed or successful action.

Reply:
ok got it.